package.json

```diff
@@ -127,5 +127,7 @@
         "typescript": "^3.9.2",
         "webpack": "^4.43.0"
     },
-    "dependencies": {}
+    "dependencies": {
```

Comment:
We have to remove this.  This dependency is already involved in the Core package.